temfpy/integration.py

```diff
@@ -74,7 +74,7 @@ def continuous(x, u, a):
     References
     ----------
     .. [G1984] Genz, A. (1984). Testing multidimensional integration routines.
-       In Proc. of international conference on Tools, methods and languages
+       In Proc. of international conference on tools, methods, and languages

```

Comment:
To remain consistent with the agreed upon reference conventions for titles, I would make Tools lower case. I would also add an Oxford comma to increase language precision.